translation/samples/generated_samples/snippet_metadata_translate_v3.json

```diff
@@ -0,0 +1,1728 @@
+{
```

Comment:
Snippetgen samples are not in scope, please remove this directory and keep it in the old repository.

Reply:
Done.